amlip_py/amlip_py/node/FiwareNode.py

```diff
@@ -285,7 +294,7 @@ def patch_inference(
 
     def get_inference(
             self,
-            timeout=0) -> dict:
+            timeout=20) -> dict:
```

Comment:
The default timeout value should remain `0`, as it indicates no timeout by default. This allows users to specify a timeout as needed for each use case.

Reply:
But in this way, the error where the fiware node is tryng to get the inference data would never occur.

Reply:
Therefore, we wouldn't be able to log it in the Dashboard.

Reply:
But in this way there would be no way to notify the Backend of the Dashboard of the error occured if there has been one, and we go back to the previous problem.

Reply:
But in this way there would be no way to notify the Backend of the Dashboard if the error occured if there has been one, and we go back to the previous problem.

Reply:
Done